ci-operator/step-registry/optional-operators/cvp-common/ibmz-eco/optional-operators-cvp-common-ibmz-eco-workflow.yaml

```diff
@@ -6,6 +6,8 @@ workflow:
       OO_INSTALL_NAMESPACE: '!create'
       OO_PACKAGE: transform-adv-operator
       OO_TARGET_NAMESPACES: '!install'
+      CUSTOM_SCORECARD_TESTCASE: ""
+      TEST_MODE: "ibm"
```

Comment:
Do we need this file in cvp-common?

PCP test ISV bundles. CVP is not involved there.